app/pages/terms/terms.less

```diff
@@ -13,52 +13,52 @@
  * not, you can obtain one from Tidepool Project at tidepool.org.
  */
 
-@terms-overlay-zindex:        @zindex-overlay-background;
-@terms-overlay-text-zindex:   @zindex-overlay;
-@terms-overlay-box-width:     800px;
-@terms-overlay-age-box-width: 500px;
+@terms-zindex:        @zindex-overlay-background;
```

Comment:
We shouldn't need these z-indices anymore since it's not an overlay anymore, right?